src/fastoad/io/variable_io.py

```diff
@@ -11,34 +11,56 @@
 #  You should have received a copy of the GNU General Public License
 #  along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+import logging
```

Comment:
Same remark as for the _LOGGER

Reply:
Fixed in a4b8a4bf354349003ee297fc5a891d1c7464f0f3